src/commcare_cloud/manage_commcare_cloud/monitors/ntp.yml

```diff
@@ -0,0 +1,31 @@
+env_key: environment.name
+id: 11866550
+message: |-
+  {{#is_alert}}
+  NTP out of sync on {{host.name}}
+  To force resolve run:
+  ```
+  service ntp stop
```

Comment:
this needs to be indented?
https://github.com/dimagi/commcare-cloud/blob/5c62a5438d62bbf67c87c967dc288488879d1b1d/src/commcare_cloud/manage_commcare_cloud/monitors/agent_check.yml#L7-L8

Reply:
why?